Submissions/JonathaWRDCosta.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "Jonatha Costa",
+  "img": "https://avatars.githubusercontent.com/u/138382592?v=4",
+  "email": "jonathawiliamsrdcosta@gmail.com",
+  "links": {
+    "website": "https://personal-portfolio-umiv.vercel.app/",
+    "linkedin": "www.linkedin.com/in/jonathawrdcosta",
+    "github": "https://github.com/JonathaWRDCosta>"
+  },
+  "jobTitle": "[Software & Data Engineer",
```

Comment:
Typo? `[`

Reply:
Okay, I think the problem in the code is fixed, and tell me if I need to do anything else. These were the steps I took

git remote add upstream git@github.com:JonathaWRDCosta/ZtM-Job-Board.git

git pull upstream master

git add .

git commit -m ""

git pull upstream master